app/pods/components/newsletter/newsletter-header-overview/component.js

```diff
@@ -64,11 +64,12 @@ export default Component.extend({
       this.set('isLoading', true);
       const agenda = await this.get('agenda');
       const meeting = await agenda.get('createdFor');
-      this.newsletterService.createCampaign(agenda, meeting).then(() => {
+      await this.newsletterService.createCampaign(agenda, meeting).then(() => {
         this.set('isLoading', false);
       });
       this.set('newsletterHTML', null);
       this.set('testCampaignIsLoading', false);
+      location.reload();
```

Comment:
Geen voorstander van de `location.reload` hier, ik snap wat je wilt doen (herladen van de view, waardoor de model hook getriggered wordt voor updatet data lijkt me) maar dit is volgens het framework waarschijnlijk niet de correcte manier. We zouden deze call moeten terugtracen naar zijn `route` en dan daar een refresh triggeren.. zij het al dan niet via een function reference over de componenten heen.

De reload is ook vrij abrupt en is een full screen refresh ipv enkel een data refresh wat je eigenlijk wilt bereiken lijkt me, als het foutloopt is de toast die geworpen wordt vrij snel weg omdat de page een refresh krijgt.

Reply:
't is simpel, het werkt (nu is het buggy)en het is stukken minder code zo dan om vannalles mee te geven naar beneden. Om het zelfde resultaat te bereiken.